### PR TITLE
feat: Remove `@@notation` from user-facing example titles

### PR DESCRIPTION
**Pull request by Josh-Cena**
Part of https://github.com/mdn/mdn/issues/562. This PR only changes user-facing strings, without touching any file paths.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
I don't really know who to request for reviews, so I'm just randomly fetching @bsmth 😄

File: live-examples/js-examples/intl/intl-prototype-@@tostringtag.js
Comment (bsmth) [inline live-examples/js-examples/intl/intl-prototype-@@tostringtag.js:1]: do we reference this in content? Same as the other deletion below

Comment (Josh-Cena) [reply live-examples/js-examples/intl/intl-prototype-@@tostringtag.js:1]: No, we don't. We removed these two pages long ago, which is why I'm simply removing the examples.

**Review by bsmth — APPROVED** [review]

**Comment by bsmth** [comment]
LGTM, thank you!

**Pull request merged**